test/groovy/CloudFoundryDeployTest.groovy

```diff
@@ -208,7 +211,8 @@ class CloudFoundryDeployTest extends BasePiperTest {
 
     @Test
     void testCfNativeAppNameFromManifest() {
-        helper.registerAllowedMethod('fileExists', [String.class], { s -> return true })
+        //helper.registerAllowedMethod('fileExists', [String.class], { s -> return true })
```

Comment:
Please remove the `helper.registerAllowedMethod` line rather than commenting it. Applies also elsewhere below.